### DISCUSSION TITLE
UniGetUI 3.1.2 beta 4

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2842] [Announcements]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.1.2-beta4/total?style=for-the-badge&color=red)

~~NOTE: This beta is expected to be the latest beta before release.
NOTE: This release is not digitally signed for convenience. Stable releases will always be signed.~~
Last beta expected to be the last prerelease before stable, yet stable will be delayed due to issues with the code signing certificate.
I was told that the issues should be fixed in a week or two, as of 08/10/2024 (dd/mm/aa)

# Changelog
 - WinGet icons will show on UniGetUI
 - Package icons will show on package lists (they can be disabled from the settings)
 - A WinGet troubleshooter has been implemented to fix when WinGet corrupts itself and stops reporting installed packages.
 - Package tags show on the bottom-right corner of the icon, instead of overriding it
 - Internal conurrency improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/UniGetUI/releases/tag/3.1.2-beta4'>UniGetUI 3.1.2 beta 4</a>.</em>

**Comment by LaKanDoR** [comment]
Improvement ideas:
1. Think about Reporting for Admins build a file db schema for presenting in a GUI Install History you could use SQLLite (local filedatabase for saving the information, searching, grouping)
2. Installed UniGet a service which can only accessed if the admin put in the GUI protection password. the service is installing fully automatically the updated (slient / not slient must be a option)
